cms/migrations/0044_external_course_cms_page.py

```diff
@@ -0,0 +1,193 @@
+                    models.CharField(
+                        help_text="A short subheading to appear below the title on the program/course page",
+                        max_length=255,
+                    ),
+                ),
+                (
+                    "video_title",
+                    wagtail.core.fields.RichTextField(
+                        blank=True,
+                        help_text="The title to be displayed for the program/course video",
+                    ),
+                ),
+                (
+                    "video_url",
+                    models.URLField(
+                        blank=True,
+                        help_text="URL to the video to be displayed for this program/course. It can be an HLS or Youtube video URL.",
+                        null=True,
+                    ),
+                ),
+                (
+                    "duration",
+                    models.CharField(
+                        blank=True,
+                        help_text="A short description indicating how long it takes to complete (e.g. '4 weeks')",
+                        max_length=50,
+                        null=True,
+                    ),
+                ),
+                (
+                    "background_video_url",
+                    models.URLField(
+                        blank=True,
+                        help_text="Background video that should play over the hero section. Must be an HLS video URL. Will cover background image if selected.",
+                        null=True,
+                    ),
+                ),
+                (
+                    "time_commitment",
+                    models.CharField(
+                        blank=True,
+                        help_text="A short description indicating about the time commitments.",
+                        max_length=100,
+                        null=True,
+                    ),
+                ),
+                (
+                    "featured",
+                    models.BooleanField(
+                        blank=True,
+                        default=False,
+                        help_text="When checked, product will be shown as featured.",
+                    ),
+                ),
+                (
+                    "content",
+                    wagtail.core.fields.StreamField(
+                        [
+                            (
+                                "heading",
+                                wagtail.core.blocks.CharBlock(classname="full title"),
+                            ),
+                            ("paragraph", wagtail.core.blocks.RichTextBlock()),
+                            ("image", wagtail.images.blocks.ImageChooserBlock()),
+                            ("raw_html", wagtail.core.blocks.RawHTMLBlock()),
+                        ],
+                        blank=True,
+                        help_text="The content of this tab on the program page",
```

Comment:
do we need this content as we won't have any link to external courses with programs?

Reply:
This is just coming in from the base `Page` model I guess.